webapp/config/packages/nelmio_api_doc.yaml

```diff
@@ -434,6 +434,28 @@ nelmio_api_doc:
                         lazy_eval_results:
                             type: boolean
                             description: Whether to use lazy evaluation for this problem. Defaults to the global setting
-
+                TeamCategoryPost:
```

Comment:
Can't we, instead of this, use the normal TeamCategory object merged with a generic ID object? I think OpenApi can do that. That saves duplication.

Reply:
It would be a negative merge though,

We need the Modal from TeamCategory minus the ID field. I'm not sure if OpenAPI can do that.

Reply:
Sorry, then do it the other way around. Create a TeamCategoryWithoutID and then a merged one